h2o-r/h2o-package/R/automl.R

```diff
@@ -282,6 +282,7 @@ h2o.automl <- function(x, y, training_frame,
     poll_state <<- do.call(.automl.poll_updates, list(job, verbosity=verbosity, state=poll_state))
   }
   .h2o.__waitOnJob(res$job$key$name, pollUpdates=poll_updates)
+  .automl.poll_updates(h2o.get_job(res$job$key$name), verbosity, poll_state) # ensure the last update is retrieved
```

Comment:
technically, I think `poll_updates` is more accurate here.
It's true that that it currently only polls the logs (the status update being polled by the `.h2o.__waitOnJob` function), but if it had to do more  things, I'd rather extract some piece of logic from the `poll_updates` function than having to rename it...

The problem comes from `__waitOnJob` here that doesn't even try to poll anything if the job is already completed. Didn't want to fix it here as it may have side effects on other parts of the R client.